lib/kitchen/provisioner/salt_solo.rb

```diff
@@ -134,27 +148,42 @@ def create_sandbox
 
       def init_command
         debug("Initialising Driver #{name} by cleaning #{config[:root_path]}")
-        cmd = "mkdir -p '#{config[:root_path]}';"
-        cmd += <<-INSTALL
-          #{config[:init_environment]}
-        INSTALL
+        if windows_os?
+          cmd = "rm ""#{config[:root_path]}"" -Recurse -Force; mkdir -Path ""#{config[:root_path]}"
```

Comment:
I can remove it so it is like the non-windows side, the root_path directory was originally recursively cleaned out for non-windows, but I did notice the change on that one while rebasing (I honestly don't have any thoughts either way if it needs to be done or not)